super-mode-calculator/src/lib/build-query.ts

```diff
@@ -0,0 +1,153 @@
+import { format, subDays, subHours } from 'date-fns';
+import { regionSql } from '../lambdas/query/regionSql';
+import {
+	SUPER_MODE_MINIMUM_AV,
+	SUPER_MODE_MINIMUM_VIEWS,
+	SUPER_MODE_WINDOW_IN_HOURS,
+} from './constants';
+import { toDateHourString, toDateString } from './date';
+
+export interface BigQueryResult {
+	url: string;
+	region: string;
+	total_av: number;
+	total_views: number;
+	av_per_view: number;
+}
+
+export const buildQueryForSuperMode = (
+	stage: 'CODE' | 'PROD',
+	now: Date = new Date(),
```

Comment:
Instead of using now, let's use `now - 1 hour` to ensure we have good data.
We did this for the bandit system here: https://github.com/guardian/support-analytics/blob/main/bandit/src/query-lambda/query-lambda.ts#L28